website/docs/cdktf/concepts/providers.mdx

```diff
@@ -5,21 +5,46 @@ description: >-
   providers in a CDK for Terraform application.
 ---
 
-# Providers
+# Providers 
 
-A [provider](/language/providers) is a Terraform plugin that allows users to manage an external API. Provider plugins like the [AWS provider](https://registry.terraform.io/providers/hashicorp/aws/latest) or the [cloud-init provider](https://registry.terraform.io/providers/hashicorp/cloudinit/latest/docs) act as a translation layer that allows Terraform to communicate with many different cloud providers, databases, and services.
+A [provider](/language/providers) is a Terraform plugin that allows users to manage an external API. In your CDK for Terraform (CDKTF) application, you use your preferred programming language to define the [resources](/cdktf/concepts/resources) you want Terraform to manage on one or more providers. 
```

Comment:
I wrote that when I first joined...but now that I have a better understanding of how things work, I believe this is more technically correct - lmk what you think!

```suggestion
A [provider](/language/providers) is a plugin that lets Terraform manage an external API. In your CDK for Terraform (CDKTF) application, you use your preferred programming language to define the [resources](/cdktf/concepts/resources) you want Terraform to manage on one or more providers. 
```